integration_test.go

```diff
@@ -375,10 +375,24 @@ func TestStop(t *testing.T) {
 }
 
 func writeDataIntoTestapp(t *testing.T, data string) {
-	err := ioutil.WriteFile("testapp/data/file", []byte(data), os.FileMode(0644))
+	tempFile, err := ioutil.TempFile(os.TempDir(), "tempData")
 	if err != nil {
-		t.Fatalf("write file: %s\n", err)
+		t.Fatalf("could not create temp file: %s\n", err)
 	}
```

Comment:
defer os.Remove(tempFile.Name())

By the doco, if the file doesn't exist, this will do nothing